src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -358,13 +363,32 @@ private CallbackResponse gaValidateMakeAnOrder(CallbackParams callbackParams) {
 
     private CallbackResponse gaValidateRequestMoreInfoScreen(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
+
         GAJudicialRequestMoreInfo judicialRequestMoreInfo = caseData.getJudicialDecisionRequestMoreInfo();
         List<String> errors = judicialRequestMoreInfo != null
             ? validateDatesForRequestMoreInfoScreen(judicialRequestMoreInfo)
             : Collections.emptyList();
 
+        if (judicialRequestMoreInfo != null
+            && SEND_APP_TO_OTHER_PARTY.equals(judicialRequestMoreInfo.getRequestMoreInfoOption())) {
+            String endDateForMoreInfoSubmission = deadlinesCalculator
+                .calculateApplicantResponseDeadline(
```

Comment:
We should not be converting date to String.... The current deadline is being saved as date but we have a ticket open to change it to Date... In similar way, all the dates including this deadline should be LocalDateTime.